labs/github/github.py

```diff
@@ -0,0 +1,79 @@
+import requests
+from labs.config import GITHUB_ACCESS_TOKEN
+from labs.config import GITHUB_API_BASE_URL
+from labs.config import GITHUB_OWNER
+from labs.config import GITHUB_REPO
+from labs.config import GITHUB_USERNAME
+from labs.config import get_logger
```

Comment:
just curious, why are all commits one liners?

Reply:
please make sure you have black properly configured in your IDE